src/shared/molecules/MyDataTable/MyDataTable.tsx

```diff
@@ -254,15 +255,17 @@ const MyDataTable: React.FC<TProps> = ({
         title: 'updated date',
         dataIndex: 'updatedAt',
         width: 140,
-        sorter: false,
+        sorter: (a, b) =>
+          moment(a.updatedAt).unix() - moment(b.updatedAt).unix(),
```

Comment:
Since `moment` has reached it's end of life, I would strongly discourage it's further use in the project, so we can gradually move away from it.

Fortunately for simple use cases, such as these the native `Date` api is already pretty good:

I think, for us this should suffice:
```javascript
new Date(a.updatedAt) - new Date (b.updatedAt)
```

if you still prefer to use ms since epoch, we can use [getTime()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime):
```javascript
new Date(a.updatedAt).getTime() - new Date (b.updatedAt).getTime()
```

Note: we might want to check for possible undefined values.